eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -85,3 +85,4 @@ steps:
       -UserReviewers "${{ parameters.GHReviewersVariable }}"
       -TeamReviewers "${{ parameters.GHTeamReviewersVariable }}"
       -Assignees "${{ parameters.GHAssignessVariable }}"
+      -CloseAfterOpenForTesting ${{ coalesce(variables['TestPipeline'], '$false') }}
```

Comment:
Should this be `false` or `$false`?

Reply:
Sorry I miss-read this statement I thought you were comparing the value, I now see you are coalesce'ing. However my question now alters to should we add the `$` outside as the TestPipeline variable likely shouldn't contain a `$`.